templates/forms/default/form.html.twig

```diff
@@ -52,6 +52,9 @@
       {% endblock %}
       {% if form.novalidate %}novalidate{% endif %}
       {% if form.keep_alive %}data-grav-keepalive="true"{% endif %}
+      {% for k, v in form.data.blueprints.form %}
```

Comment:
I would prefer to have something like 

```yaml
name:
  attributes:
    data-foo: value
   onClick: value
```
As you can see, this would allow you to add any attribute, not just the data ones and is far more future proof.

@rhukster @w00fz what do you think?

Reply:
The data nesting structure is quite complex (especially wrt `blueprints` that I want to avoid looking at as much as possible), I don't know how`attributes` would support `@` (handy) magic values and I'm clearly not able to provide and extensively tested PR for this.

I already have many forms depending on custom attributes and for which I'm strugling with custom patches for more than a year. So I'm fine with anything you'd finally merge.
But I just can't provide you a thoroughly tested `attributes`-based PR anymore.

Reply:
Here is a new PR. It resolves your concerns and use your approach.
Sadly it lost the blueprint expansion feature (which, I think, is quite a miss, but I'm unable to dig-out why and how to make it work).
Feel free to amend or take-over.